codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerOperationHandlerGenerator.kt

```diff
@@ -132,13 +134,18 @@ class ServerOperationHandlerGenerator(
         } else {
             symbolProvider.toSymbol(operation.outputShape(model)).fullName
         }
+        val streamingBodyTraitBounds = if (operation.inputShape(model).hasStreamingMember(model)) {
+            "\n B: Into<#{SmithyHttpServer}::ByteStream>,"
```

Comment:
You can remove the `\n` and interpolate the variable in the next line.

Reply:
@david-perez the reason I did this was that the output is usually not including this streaming member and then it generates an empty line, which is not causing any compilation issues, but is weird to watch...

Reply:
We run "cargo fmt" at the end of the codegen, so the weird look should go away.. Is it not?

Reply:
Nope, the empty line is still there. Perhaps because it's just an empty line then cargo fmt will not remove it.

Reply:
is there a better way to generate an optional line?

Reply:
Not that I know of. Maybe @rcoh knows.